lib/iris/fileformats/abf.py

```diff
@@ -118,9 +118,9 @@ def to_cube(self):
 
         # Name.
         if self.format.lower() == "abf":
-            cube.rename("leaf_area_index")
-        elif self.format.lower() == "abl":
             cube.rename("FAPAR")
```

Comment:
FWIW it took me some searching to convince myself that FPAR and FAPAR are names for the same thing.
But I think I can confirm that now, as [this paper](https://www.frontiersin.org/articles/10.3389/fpls.2018.00776/full) references [this one](https://www.sciencedirect.com/science/article/pii/S0034425798000509?via%3Dihub), apparently assuming they are the same.